health/notifications/health_alarm_notify.conf

```diff
@@ -212,8 +212,27 @@ DEFAULT_RECIPIENT_EMAIL="root"
 # to corelate messages and also will cause most clients to group all the
 # messages together.  This is enabled by default, uncomment the line
 # below if you want to disable it.
-#EMAIL_THREADING="NO"
+#EMAIL_THREADING="YES"
 
+#------------------------------------------------------------------------------
+# hangouts (google hangouts chat) global notification options
+
+# multiple recipients can be given like this:
+#                  "CHANNEL1 CHANNEL2 ..."
+
+# enable/disable sending hangouts notifications
+SEND_HANGOUTS="YES"
+
+# Login to hangouts and create an incoming webhook. You need only one for all
+# your netdata servers (or you can have one for each of your netdata).
+# Without it, netdata cannot send hangouts notifications.
+HANGOUTS_WEBHOOK_URL=""
+
+# if a role's recipients are not configured, a notification will be send to
+# this hangouts channel (empty = do not send a notification for unconfigured
+# roles):
+#DEFAULT_RECIPIENT_HANGOUTS="<Name of the channel>"
```

Comment:
This should also go to README.md file

Reply:
@paulfantom I asked for this.  I missed bit before it mentioning that the recipients are channels.

@EnzoAkira Sorry about this one, I missed the stuff before it.  You can remove this commented line.

Reply:
@Ferroin No problem.